adapters/buenos-aires.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+import cheerio from 'cheerio';
+import async from 'async';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+
+export const name = 'buenos aires';
+export function fetchData (source, callback) {
+  // list of requests used to get data
+  const tasks = generateTasks(source);
+
+  async.parallel(tasks, (err, response) => {
+    if (err) {
+      return callback(new Error('Failure to load data urls.'));
+    }
+
+    // wrap in try catch to handle possible errors
+    try {
+      const data = formatData(response);
+      if (data === undefined) {
+        return callback(new Error('Failure to parse data.'));
+      }
+      callback(null, data);
+    } catch (e) {
+      return callback(new Error('Unknown adapter error.'));
+    }
+  });
+}
+
+// makes a requests list of requests for each station to each pollutant
+const generateTasks = (source) => {
+  const linkRequests = [];
+  ['CENTENARIO', 'CORDOBA', 'LA BOCA'].forEach((station) => {
+    ['CO', 'NO2', 'PM10'].forEach((pollutant) => {
+      const makeLinks = (station, pollutant) => {
+        const day = moment().date().toString();
+        const month = '0' + (moment().month() + 1).toString();
+        const year = moment().year().toString();
+        let stationNum;
+        switch (station) {
+          case 'LA BOCA':
+            stationNum = 1;
+            break;
+          case 'CENTENARIO':
+            stationNum = 2;
+            break;
+          case 'CORDOBA':
+            stationNum = 3;
+            break;
+          default:
+            break;
+        }
+        let pollutantNum;
+        switch (pollutant) {
+          case 'CO':
+            pollutantNum = 1;
+            break;
+          case 'NO2':
+            pollutantNum = 2;
+            break;
+          case 'PM10':
+            pollutantNum = 3;
+            break;
+          default:
+            break;
+        }
+        const link = [
```

Comment:
Can you do this as a string template literal? This just reads as a bit confusing to me.